internal/account/accessor.go

```diff
@@ -0,0 +1,52 @@
+package account
+
+import (
+	"context"
+	"kg/procurement/internal/common/database"
+
+	"github.com/benbjohnson/clock"
+)
+
+const (
+	insertAccountQuery = `
+		INSERT INTO account (
+			id,
+			email,
+			password,
+			modified_date
+		) VALUES (
+			$1,
+			$2,
+			$3,
+			$4
+		)
```

Comment:
how about using `NamedExec` just to increase clarity, also IMO it looks nicer that way xD

Reply:
Yes, I wanted to use `NamedExec` but unfortunately It's not supported with `ExpectExec` in the unit test

Reply:
Revision: I just noticed, we CAN use it. I've refactor my code to use it, please rereview 🙇‍♂️